pkg/network/encoding/encoding.go

```diff
@@ -83,6 +84,13 @@ func modelConnections(conns *network.Connections) *model.Connections {
 			"detected orphan http aggreggations. this can be either caused by conntrack sampling or missed tcp close events. count=%d",
 			httpEncoder.orphanEntries,
 		)
+
+		telemetry.NewMetric(
+			"http.orphan_aggregations",
+			telemetry.OptMonotonic,
+			telemetry.OptExpvar,
+			telemetry.OptStatsd,
+		).Add(int64(httpEncoder.orphanEntries))
```

Comment:
my concern is a new developer will have a struggle to understand all metrics we have in the agent
some metrics are defined in a single files, another metrics is being declared inside a function
imho, i think we should move the declaration of the metric (and all other metrics) into a single file

Reply:
Gotcha, I think that's a fair concern, but I opted for _partially_ addressing this by exposing a `/network_tracer/debug/telemetry` endpoint which will list all metrics currently being tracked by the library.
moreover the code discoverability aspect of metrics is not that bad now because they're all constructed using the `NewMetric` method.

Having said that, I do see some value in having the single file declaration as you said, because the metric above for example would only be listed in the debug endpoint after that code path is reached (which takes 30s in this case :disappointed: ), but there are also cases where this approach would not fit well, such as "dynamic" metrics (eBPF map errors, for example, or anything else that you can't or it's just to tricky to declare beforehand)

Considering all above, I'd vote for keeping things as they're for now, and try this for a little bit longer before doing any further changes or standardizing this is any way.